zeth_contracts/contracts/BLS12_377_test.sol

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+pragma solidity ^ 0.5.0;
+
+contract BLS12_377_test
+{
+    // In many cases, these numbers must be used as literals in the assembly
+    // code.
+
+    uint256 constant scalarWords = 1;
+    uint256 constant scalarBytes = scalarWords * 32; // 32 (0x20)
+
+    uint256 constant g1CoordWords = 2;
+    uint256 constant g1CoordBytes = g1CoordWords * 32; // 64 (0x40)
+    uint256 constant g1PointWords = 2 * g1CoordWords;
+    uint256 constant g1PointBytes = g1PointWords * 32; // 128 (0x80)
+
+    uint256 constant g2CoordWords = 4;
+    uint256 constant g2CoordBytes = g2CoordWords * 32; // 128 (0x80)
+    uint256 constant g2PointWords = 2 * g2CoordWords;
+    uint256 constant g2PointBytes = g2PointWords * 32; // 256 (0x100)
+
+    /// `points` should be the concatenation of 2 encoded points
+    function testECAdd(bytes32[2 * g1PointWords] memory points) public returns(
+        bytes32[g1PointWords] memory)
+    {
+        bytes32[g1PointWords] memory output;
+        bool success = true;
+        assembly
+        {
+        success := call(gas, 0xc4, 0, points, 0x100, output, 0x80)
```

Comment:
Missing indentation (also applicable to files below)

Reply:
indented all assembly blocks